### DISCUSSION TITLE
💡feat:  Updating Home Page Style

### DISCUSSION THREAD

**Discussion by vedprakashnautiyal** [priyankarpal/projectshut#125] [Ideas]
# Problem
I think all the social related to sharing this project should be at one place, rather than at two different places on the website (header and footer)

# Solution
Star on github, share on twitter and share on linkedin (if introduced) should be at the top for the first time visitors to see

# Here are some suggested changes
<br>

## Current Look
<img width="1680" alt="Old Head" src="https://user-images.githubusercontent.com/71223796/228909924-ddf3a64d-dfac-4529-b26d-0805c3296f40.png">
<img width="1680" alt="Old Foot" src="https://user-images.githubusercontent.com/71223796/228909948-7782ea3c-c499-4774-8462-656e83e8e350.png">
<br>

## New Look
<img width="1680" alt="New Head" src="https://user-images.githubusercontent.com/71223796/228909992-8fdc2bb0-5f4c-4cb9-893f-840615073094.png">
<img width="1680" alt="New Foot" src="https://user-images.githubusercontent.com/71223796/228910006-f1284ca6-d2fe-4bcb-b020-f97d1c5a1587.png">

**Comment by priyankarpal** [comment]
yes. +1

could you share the mobile view?

**Comment by priyankarpal** [comment]
also, we can use only one button for sharing.

**Comment by priyankarpal** [comment]
ok, I'm moving this to a discussion.

**Comment by priyankarpal** [comment]
Completed.
